spec/adapters/highcharts_spec.rb

```diff
@@ -397,8 +397,8 @@
       html = File.read(path)
       expect(html).to match(/html/)
       expect(html).to match(/Chart/)
-      expect(html).to match(/BEGIN highstock.js/i)
-      expect(html).to match(/END highstock.js/i)
+      expect(html).to match(/BEGIN js\/highstock.js/i)
```

Comment:
Let us know why it is under `js` folder now.

Reply:
There are two dependencies:
1. http://code.highcharts.com/stock/highstock.js
2. http://code.highcharts.com/stock/js/highstock.js
Former is used in classic mode while the latter one+CSS dependency is used in styled mode which I have implemented in web frameworks (so, I have put this dependency separately in js folder).